Feature/initial mode sku (#625)
vtex-apps/store-components: CHANGELOG.md

and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [Unreleased]
### Added
- `initialSelection` and `showVariationsErrorMessage` to `SKUSelector`.

> -  `showTooltipOnSkuNotSelected`to `BuyButton`.

Reviewer: ```suggestion
-  `showTooltipOnSkuNotSelected` to `BuyButton`.
```

> -  `showTooltipOnSkuNotSelected` to `BuyButton`.

Reviewer: ```suggestion
- `showTooltipOnSkuNotSelected` to `BuyButton`.
```

### Changed

> - `BuyButton` now only add a product to the cart if all variations have one options selected.

Reviewer: ```suggestion
- `BuyButton` now only adds a product to the cart if all variations have one option selected.
```
